plugins/discourse-calendar/spec/lib/discourse_post_event/rrule_generator_spec.rb

```diff
@@ -33,6 +33,47 @@
     end
   end
 
+  context "when showLocalTime is enabled" do
```

Comment:
```suggestion
  describe "show_local_time" do
```

Since you are testing enabled in one `it` and disabled in the ohter